### ISSUE TITLE
How to make a Plugin for my own website?

### ISSUE THREAD

**Issue by ProbirBiswas** [YahnisElsts/plugin-update-checker#316]
How to make a Plugin for my own website? my website sale GPL license theme and plugin I make a plugin like [GPLkit](https://www.gplkit.com/) or [Gplzone ](https://gplzone.com/automatic-updates/) and like [GPL Vault](https://www.gplvault.com/how-to-use-install-the-gpl-vault-automatic-update-plugin/) for automatically update my customer.

**Comment by YahnisElsts** [comment]
Sorry, I'm not sure I understand the question. Could you rephrase it?

If you're asking how to make the update checker work with your plugin, the [Getting Started](https://github.com/YahnisElsts/plugin-update-checker#self-hosted-plugins-and-themes) section provides the basic instructions. As for the server-side component, you could use [wp-update-server](https://github.com/YahnisElsts/wp-update-server), but if you want to support stuff like accounts and API keys you will probably need to do quite a bit of work to add those features. It doesn't support anything like that right now.
